roles/setup_etcd/defaults/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+# defaults file for setup_etcd
+os: ""
+
+pg_type: "PG"
+pg_version: 14
+pg_owner: "postgres"
+pg_group: "postgres"
+
+use_hostname: true
+
+etcd_version: "3.5.6"
+etcd_service: "etcd-{{ etcd_version }}"
+etcd_download_url: "https://github.com/etcd-io/etcd/releases/download/v{{ etcd_version }}"
+etcd_checksum_url: "sha256:https://github.com/coreos/etcd/releases/download/v{{ etcd_version }}/SHA256SUMS"
+etcd_architecture: "amd64"
+etcd_package: "etcd-v{{ etcd_version }}-linux-{{ etcd_architecture }}"
+etcd_rpm: false
+
+etcd_name: "{{ inventory_hostname }}"
+etcd_client_port: 2379
+etcd_peer_port: 2380
+etcd_compaction_mode: revision
+etcd_compaction_retention: 10
+etcd_initial_cluster: []
+etcd_initial_cluster_state: "new"
+etcd_cluster_nodes: []
+etcd_data_dir: "/var/lib/etcd"
+etcd_config_dir: "/etc/etcd"
+etcd_config: "{{ etcd_config_dir }}/etcd-{{ etcd_version }}.conf"
+etcd_owner: "{{ pg_owner }}"
```

Comment:
From a security point of view, it's better to have a dedicated user for running the etcd service